help/destinations/catalog/crm/zendesk.md

```diff
@@ -175,9 +186,11 @@ This section captures the functionality and significant documentation updates ma
 
 |Release month|Update type|Description|
 |---|---|---|
+|November 2023|Functionality update|<ul><li>We now automatically convert the **[UICONTROL Mapping ID]** values in the [schedule audience export](#schedule-segment-export-example) step to `lowercase`. Additionally we also sanitize `spaces` and `special characters`. This is done to ensure compliance with [!DNL Zendesk] field names.</li><li>We now support custom uploads, see the [supported audiences](#supported-audiences) section.</li></ul>|
```

Comment:
General docs feedback moving forward: always talk about what the user can do with a feature/change/update. Not about what we/Adobe did.

Also, `code` markup should only be used for variables and actual code. Not for " `spaces` and `special characters`".

Let's rephrase this as shown below.

|Release month|Update type|Description|
|---|---|---|
|November 2023|Functionality update|<ul><li>You can now activate external audiences originating from custom uploads to this destination. See the [supported audiences](#supported-audiences) section for more details.</li><li>The **[UICONTROL Mapping ID]** values in the [schedule audience export](#schedule-segment-export-example) step are now automatically converted to lowercase values. Additionally, spaces and special characters are now automatically sanitized to ensure compliance with the [!DNL Zendesk] field names.</li></ul>|